retroravelry/src/test/java/com/caseykulm/retroravelry/RavelryClientTest.kt

```diff
@@ -12,4 +12,10 @@ class RavelryClientTest {
     println(libResp)
     assertNotNull(libResp)
   }
+
+  @Test
+  fun searchPatternsShouldNotBeNull() {
+    val searchResponse = ravelryClient.searchPatterns("cardigan", 1, 20)
```

Comment:
We should subscribe to this with a `TestSubsciber` by using, 

```kotlin
val testSub = searchResponse.test()
testSub.assertNoErrors()
```

instead of the null check

Reply:
`searchResponse.blockingFirst()` seems useful here instead actually :D